app/controllers/state_file/questions/nj_household_rent_own_controller.rb

```diff
@@ -17,7 +17,7 @@ def next_path
         when 'own'
           NjHomeownerEligibilityController.to_path_helper(options)
         when 'both'
-          NjUnsupportedPropertyTaxController.to_path_helper(options)
+          NjIneligiblePropertyTaxController.to_path_helper(options)
```

Comment:
[DUST] Support for "Both" is being redone in a separate ticket https://github.com/newjersey/affordability-pm/issues/192